src/components/MovieDescription/MovieDescription.js

```diff
@@ -1,20 +1,35 @@
 import './MovieDescription.css';
+import Genre from '../Genre/Genre';
 
 const MovieDescription = ({info}) => {
+    const formatDollarString = num => {
+        if (num >= 100000000) {
+            return `$${(num/100000000).toFixed(1)} billion`
+        }
+        if (num >= 1000000) {
+            return `$${(num/1000000).toFixed(1)} million`
+        }
+        return `$${num}`
```

Comment:
I love this! I was thinking about trying to add commas with JS but this looks wayyyyy better!